### ISSUE TITLE
Resource name when Model is namespaced 

### ISSUE THREAD

**Issue by nenads** [CrestApps/laravel-code-generator#78]
### Environment:
- Laravel-Code-Generator Version: 2.3.#
- Laravel Version: 5.6.#

### Content Of The Resource-File:

![image](https://user-images.githubusercontent.com/1824176/44050193-9127ddb8-9f36-11e8-8310-490456c5cce0.png)

**Comment by nenads** [comment]
maybe this is related to my test fix  #77

**Comment by CrestApps** [comment]
You are using the command incorrectly. The `resource-file:from-database Cpu` accepts ModelName as the first argument. This command does not really care about the path of the model.

So you're command should look like this instead
```
php artisan resource-file:from-database Cpu --table-name=device_cups
```

When you're getting ready to create the resource, you can change the default namespace of your model like this
```
php artisan create:scaffold Cpu --model-directory="Devices" 
```

Not the above command will create a folder called `Devices` inside the default models path which is by default [`App\Models` ](https://github.com/CrestApps/laravel-code-generator/blob/v2.3/config/default.php#L166-L172)
